staff/ana-sanjuan/mailchimp/index.html

```diff
@@ -0,0 +1,140 @@
+                </section>
+                <span class='main-section__triangle'><input class="main-section__button" type="button" value='Learn More'></span>
+            </section>
+            <section class='main-section__news'>
+                <div><img src="mailchimp/images/home/denim2.jpg" alt="example image"> </div>
+                <section>
+                <h5>Customer Story:</h5>
+                <h3>Raleigh Denim</h3>
+                <p class='main-section__p'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi culpa voluptatum omnis saepe sequi architecto nam provident odio molestiae quos eos animi, eaque, nisi, doloribus aliquid suscipit incidunt vitae vero!</p>
+                </section>
+                <input class="main-section__button" type="button" value='Learn More'>
+            </section>
+            <section class='main-section__news'>
+                <div><img src="mailchimp/images/home/unfurlr-thumb.jpg" alt="example image"> </div>
+                <section>
+                <h5>App Spotlight:</h5>
+                <h3>Unfurl</h3>
+                <p class='main-section__p'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi culpa voluptatum omnis saepe sequi architecto nam provident odio molestiae quos eos animi, eaque, nisi, doloribus aliquid suscipit incidunt vitae vero!</p>
+                </section>
+                <input class="main-section__button" type="button" value='Learn More'>
+            </section>
+            <section class='main-section__news'>
+                <div><img src="mailchimp/images/home/golden-monkeys.jpg" alt="example image"> </div>
+                <section>
+                <h5>App Spotlight:</h5>
+                <h3>Golden Monkeys</h3>
+                <p class='main-section__p'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi culpa voluptatum omnis saepe sequi architecto nam provident odio molestiae quos eos animi, eaque, nisi, doloribus aliquid suscipit incidunt vitae vero!</p>
+                </section>
+                <input class="main-section__button" type="button" value='Learn More'>
+            </section>
+        </div>
+    </div>
+    </main>
+
+    <footer>
+        <section class='upper-footer'>
+            <section class='upper-footer__about'>
+                <h4>About Us</h4>
+                <ul>
+                    <li>About MailChimp</li>
+                    <li>Jobs</li>
+                    <li>Customer Stories</li>
+                    <li>Press Resources</li>
+                    <li>Press Realises</li>
+                    <li>Integration Fund</li>
+                    <li>Brand Assets</li>
+                </ul>
+            </section>
+
+            <section class='upper-footer__connect'>
```

Comment:
Some of the classes you created in index are not used for the css stylesheet, in that case I guess you could avoid them. If, not you could maybe put some more general classes and repeat them in more than one tag, like this you would reuse css code. You have done it using inheritance % in sass or using some tags instead of classes in sass, but you could change it to be more BEM oriented.

Reply:
ok! yes, I will check the classes I don´t need and bemify them